ocrd_tesserocr/common.py

```diff
@@ -66,12 +172,18 @@ def image_from_page(workspace, page,
                  'y': 0,
                  'w': page_image.width,
                  'h': page_image.height}
+    # FIXME: uncomment as soon as we get @orientation in PageType:
```

Comment:
We should have this by now.

Reply:
@kba What is the status of the PAGE 2019 issue?

Reply:
To get @orientation we need PAGE 2019. But as we discussed yesterday, I will wait with that until @bertsky has time to formulate his idea on consistent URL for PAGE with XML version mechanism. Until then, use 2018 throughout, e.g. https://github.com/OCR-D/format-converters/pull/5